URL Detection

```diff
@@ -0,0 +1,3 @@
+NetworkCommunicationEvents  
+| where RemoteUrl == "www.microsoft.com"
```

Comment:
Are you sure you only want `==`? Maybe `has` is will be better, because it will cause URLs that actually contains the domain?